alpha/veneer/semver/README.md

```diff
@@ -0,0 +1,222 @@
+**Description of the change:**
+
+  Since a `veneer` is identified as an input schema which may be processed to generate a valid FBC, we can define a `semver veneer` as a schema which uses channel conventions to facilitate the auto-generation of channels along `semver` delimiters.
+  
+  ***Schema Anatomy***  
+    This veneer type currently supports channel names `candidate`, `fast`, and `stable`. The `stable` channel is required.
+    `generatemajorchannels` and `generateminorchannels` dictate whether this veneer will generate X-stream- or Y-stream-based channels. (Both attributes can be set simultaneously.)
+    Under each channel are a list of bundle image references which contribute to that channel.  
+
+With the (functional) example:
+```yaml
+schema: olm.semver
+generatemajorchannels: true
+generateminorchannels: true
```

Comment:
TIL that Go's json decoder is case insensitive wrt field names.

I'm +1 on camel casing these in our docs though. That seems to be the standard in YAML (k8s-related YAML at least), and it would align with the JSON tags on the struct.

Reply:
Done.